phpseclib/File/X509.php

```diff
@@ -4407,7 +4402,7 @@ function unrevoke($serial)
      */
     function getRevoked($serial)
     {
-        if (is_array($rclist = $this->_subArray($this->currentCert, 'tbsCertList/revokedCertificates'))) {
+        if (is_array($rclist = &$this->_subArray($this->currentCert, 'tbsCertList/revokedCertificates'))) {
```

Comment:
These changes will probably need looking at more closely. I have those inline statements, btw. They can probably be easily removed too.